browser-polyfill.js

```diff
@@ -407,7 +407,7 @@
             "maxArgs": 2
           }
         },
-        "pageAction": {
+        "Action": {
```

Comment:
Modifying this file is less than ideal because this is library code. The polyfill does not support MV3.
Moreover, it might not be needed anymore since chrome now supports many promise based APIs.
Try replacing this file with:
```js
const global = typeof globalThis !== 'undefined' ? globalThis : typeof self !== 'undefined' ? self : this
global.browser = typeof browser === 'undefined' ? chrome : browser
```